sdk/nodejs/lambda/lambdaMixins.ts

```diff
@@ -406,13 +406,11 @@ export class CallbackFunction<E, R> extends LambdaFunction {
         // Copy over all option values into the function args.  Then overwrite anything we care
```

Comment:
Can't put the comment on that line because it's outside of the PR, but we should also update the docs to reflect the new default: https://github.com/pulumi/pulumi-aws/blob/7461d6834b90936a38e80cc4bac2c10ff97499a0/sdk/nodejs/lambda/lambdaMixins.ts#L227